include/valkey/valkey.h

```diff
@@ -154,17 +158,21 @@ struct valkeySsl;
 
 #define VALKEY_OPT_NONBLOCK 0x01
 #define VALKEY_OPT_REUSEADDR 0x02
-#define VALKEY_OPT_NOAUTOFREE 0x04        /* Don't automatically free the async
-                                          * object on a connection failure, or
-                                          * other implicit conditions. Only free
-                                          * on an explicit call to disconnect()
-                                          * or free() */
-#define VALKEY_OPT_NO_PUSH_AUTOFREE 0x08  /* Don't automatically intercept and
-                                          * free RESP3 PUSH replies. */
-#define VALKEY_OPT_NOAUTOFREEREPLIES 0x10 /* Don't automatically free replies. */
-#define VALKEY_OPT_PREFER_IPV4 0x20       /* Prefer IPv4 in DNS lookups. */
-#define VALKEY_OPT_PREFER_IPV6 0x40       /* Prefer IPv6 in DNS lookups. */
-#define VALKEY_OPT_PREFER_IP_UNSPEC (VALKEY_OPT_PREFER_IPV4 | VALKEY_OPT_PREFER_IPV6)
+
+/* Don't automatically free the async object on a connection failure, or
```

Comment:
To be honest, I didn't manage to properly format it, for some reason even the formatter itself gets quite intrusive, like it breaks the string
```c
#define VALKEY_OPT_PREFER_IPV6 0x40 /* Prefer IPv6 in DNS lookups. */
```
to something like
```c
#define VALKEY_OPT_PREFER_IPV6                                  \
    0x40  /* Prefer IPv6 in DNS lookups. */
```

which IMHO is very ugly and I couldn't find a setting to prevent this from happening